docs/reference/errors-and-warnings/NU3042.md

```diff
@@ -0,0 +1,37 @@
+---
+title: NuGet Warning NU3042
+description: NU3042 warning code
+author: dtivel
+ms.author: dtivel
+ms.date: 03/22/2023
+ms.topic: reference
+ms.reviewer: 
+f1_keywords: 
+  - "NU3042"
+---
+
+# NuGet Warning NU3042
+
+*NuGet 6.6.0+ on Linux and macOS only*
+
+<pre>The following X.509 root certificate is untrusted because it is not present in the certificate bundle at &lt;file-path&gt;.  For more information, visit https://aka.ms/nuget/NU3042.
```

Comment:
```suggestion
<pre>The following X.509 root certificate is untrusted because it is not present in the certificate bundle at &lt;file-path&gt;.
```

The idea with error codes would be that the warning message wouldn't need the extra link. 
NU3042 should be discoverable.

Reply:
@nkolev92, I think I resolved this satisfactorily.  Please verify.

Note that I will update the resource string in the NuGet.Client PR separately.